README.md

```diff
@@ -64,6 +64,25 @@ PolicyGenius specific tips
   * Then, run `brew search elasticsearch` to see a current list of Elasticsearch versions and install `elasticsearch@2.4` through Homebrew with `brew install elasticsearch@2.4`.
```

Comment:
should we delete this `elasticsearch@2.4` messaging? or leave it as is?

Reply:
yeah we can probably remove this message